modules/ROOT/pages/faq.adoc

```diff
@@ -336,3 +336,44 @@ systemd:
 
 For more information see
 https://github.com/coreos/fedora-coreos-tracker/issues/519[the tracker issue discussion].
+
+== After the rebase to Fedora 33 why does my tooling that uses SSH cease to work?
+
+In Fedora 33 there was a change to
+https://www.fedoraproject.org/wiki/Changes/StrongCryptoSettings2[implement stronger crypto defaults].
+Part of this included taking the
+https://www.openssh.com/txt/release-8.3[advice of OpenSSH] upstream
+and disabling the use of the `ssh-rsa` public key signature algorithm.
+
+You may hit issues if you use RSA keys and:
+
+* use an old version of the `SSH` client
+* use tooling/software libraries that don't support using RSA SHA2 host key signatures
+
+For example, GoLang has an https://github.com/golang/go/issues/37278[open issue]
+to solve this problem in the GoLang stack, but has yet to resolve the
+issue. This has been hit and worked around by the FCOS community in
+our build tooling and also our higher level projects:
+
+- https://github.com/coreos/fedora-coreos-tracker/issues/699[coreos/fedora-coreos-tracker#699]
+- https://github.com/coreos/coreos-assembler/issues/1772[coreos/coreos-assembler#1772]
+
+If you run into this problem and need to workaround the issue you
+have a few options:
+
+- Switch to a newer non-RSA key type.
```

Comment:
```suggestion
- Switch to a newer non-RSA SHA-1 key type.
```

(RSA SHA-2 works, right?)

Reply:
The RSA SHA2 keyex algo works, but using a non RSA key itself will workaround the problem entirely, which is what I'm trying to say here.